python/src/spark_rapids_ml/core.py

```diff
@@ -696,60 +696,74 @@ def fitMultipleModels() -> List["_CumlModel"]:
         else:
             return super().fitMultiple(dataset, paramMaps)
 
-    def _try_stage_level_scheduling(self, rdd: RDD) -> RDD:
-        ss = _get_spark_session()
-        sc = ss.sparkContext
+    def _skip_stage_level_scheduling(self, spark_version: str, conf: SparkConf) -> bool:
```

Comment:
Any reason to ask the caller to provide the version and conf vs. just passing the sparkContext?  Seems weird/inefficient.  Saw that this is used _somewhat_ in tests, so maybe find a way to mock the version instead for testing?

Reply:
yeah. It's for testing purposes.